### DISCUSSION TITLE
Innovation Backlog App Isolation

### DISCUSSION THREAD

**Discussion by jayaro-work** [microsoft/powerapps-tools#998] [Q&A]
First of all thanks for all the great work!

Several of our departments want to use the Backlog App, and we were wondering what would be the easiest way to segregate data. I initially thought in creating separate apps, and add a default field that gets added to the submission and that way we can filter to show only certain values. Second, option would be to use separate environments but it seems overkill, or third to deploy them under teams, but unsure if the Dataverse is segregated by "Team" or by Tenant

However, just wanted to double check if there is any other approach that might be easier, or if there would be a feature in the future that would allow for this.

**Comment by manuelap-msft** [comment]
Good question! So far we have seen the Environment approach where the solution is installed in separate environments dedicated to that specific business area - that can be either Production environments or Dataverse for Teams environments. 

Another option would be to use Dataverse Security Roles and Business Units to manage who can access what part of the data - for this you would have to set up the Business Units and Teams in one environment, and change security roles so that people can only read/write data "for their business unit". This should work, but will require some initial setup and maintenance on your side.

Thank you for using the CoE
Manuela
